### DISCUSSION TITLE
Drop Cracked Account Support

### DISCUSSION THREAD

**Discussion by StormyIceLeopard** [NamelessMC/Nameless#2222] [Ideas]
By even allowing cracked servers/accounts to even work with this CMS you are prompting pirating. It is also in violation of Microsoft's EULA, which bought Minecraft.

**Comment by Derkades** [comment]
We do not condone piracy and generally assume people are using legitimate Minecraft accounts when they use Minecraft integration features. For example, skins do not work for cracked servers and the Spigot plugin does not support cracked servers.

Could you explain how you think NamelessMC explicitly supports cracked accounts?

**Reply by StormyIceLeopard** [reply]
![image](https://user-images.githubusercontent.com/10292779/112767364-608b7600-8fe4-11eb-8e15-f4bdf4e68852.png)
Why would you need these if you were not supporting cracked accounts?
I am failing to see a valid use for this for valid paid Minecraft accounts.

**Reply by Derkades** [reply]
Both these systems could be used to ensure users on the website own a Minecraft account with the username they registered with, regardless of whether the server is cracked or not

**Reply by StormyIceLeopard** [reply]
Forgive me, I am still failing to see the point.

**Comment by samerton** [comment]
As @Derkades says, we don't support cracked servers.

The two features highlighted in the screenshot have valid usages for premium servers.

One potential use case for AuthMe is for servers that want an extra layer of verification to protect their servers against compromised admin accounts.

If a web server is unable to communicate with Mojang's API to retrieve a UUID, for example the Mojang API is down for maintenance, website staff can disable account verification temporarily to allow registrations to continue.
